### ISSUE TITLE
Set Radio to split when clicked from Cluster

### ISSUE THREAD

**Issue by vazquezrodjoel** [foldynl/QLog#506]
Could it be possible to setup the radio in split right from the DX Cluster double click for obvious messages.  I know there might be a lot to cover there, but at least for example just for those "Up 5", or "1 up", or "listening 28.510".

**Comment by foldynl** [comment]
This involves several steps. The first is implementing support for split mode (something that has been on the TODO list for a long time). The second part is figuring out how to reliably detect "Up," since everyone writes it differently. Lastly, I’m not a CW operation specialist, but do we really want all operators to click on a spot and all are tuned to "Up 5"? That probably wouldn’t be a very desirable feature. But I'll leave that decision to the CW specialists. I should also add that QLog is a logging application, and every feature needs to be carefully considered to determine whether it should be implemented in QLog or in another application. QLog should not be an interface between an operator and and his/her rig. QLog should help you  record QSO parameters.

**Comment by vazquezrodjoel** [comment]
Exactly,  everyone writes it different, but for the most part they mention either "UP #" for example for cw "up 1" or "UP 2";  or for SSB "UP 5" or "UP 10".  

if they just mention "listening up" it usually means 1 in cw or 5 on SSB.  But I would just not try to cover ALL the cases, just the basics and obvious.  like the first examples, cw "up 1" or "UP 2";  or for SSB "UP 5" or "UP 10", those type of situation, and those that specify the specific frequency.  Keep it simple and improve it in future releases if needed after a "basic" setup is implemented.
